index.html

```diff
@@ -0,0 +1,183 @@
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="see-project">See project </button>
+                </div> 
+            </div>
+            <div class="project2">  
+                 <div class="project-detail">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="see-project">See project </button>
+                </div> 
+            </div>
+            <div class="project3">  
+                 <div class="project-detail">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                         <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                     <button class="see-project">See project </button>
+                 </div> 
+            </div>
+            <div class="project4">  
+                 <div class="project-detail">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="see-project">See project </button>
+                </div> 
+            </div>
+             <div class="project5">  
+                <div class="project-detail">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="see-project"> <a href="https://github.com/hiwina"></a> See project </button>
+                </div> 
+            </div>
+            <div class="project6">  
+                <div class="project-detail">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <ul class="button-detail"> 
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavScript</li>
+                        <li>html</li>
+                    </ul>
+                    <button class="see-project">See project </button>
+                </div> 
+            </div>
+        </section>
+    </section>
+    <section class="about back-image-2">
```

Comment:
- [ ] Kindly have a look at the margins/ padding used for the whole about me section. If you look keenly, this section is not fixed in the middle as all other sections are and it looks like the margin/ padding-left is greater than the margin/ padding-right. Kindly resolve this to have a consistent design all through.

[Screencast from 28-10-2022 04:35:49 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/198616150-53eaa4d1-7ec4-40b7-8ba7-04b20ee25e45.webm)